api/files/serializers.py

```diff
@@ -88,6 +111,7 @@ class FileSerializer(JSONAPISerializer):
         'size',
         'provider',
         'last_touched',
+        'tags',
```

Comment:
There is an issue to do that, but yeah, not for this. Okay, could you add the indices?